subtitles/en/csfg_human_computer_interaction_intro.vtt

```diff
@@ -47,98 +48,98 @@ Yes I would like a lemonade.
 Yes!
 
 00:38.200 --> 00:42.600
-<i>Five lemonades coming up!</i>
+Five lemonades coming up!
 
 00:43.000 --> 00:47.000
 Have you ever tried setting the time on one of these?
 
 00:55.200 --> 00:58.000
-<i>How many napkins would you like?</i>
-<00:57.000>Ah, four please.
+How many napkins would you like?
+Ah, four please.
 
 00:58.000 --> 00:59.000
-<i>One?</i>
+One?
 No.
 
 00:59.000 --> 01:00.100
-<i>Two.</i>
+Two.
 No.
 
 01:00.100 --> 01:01.200
-<i>Three.</i>
+Three.
 No.
 
 01:01.200 --> 01:03.200
-<i>Four.</i>
-No.<01:02.000> Wait yes!
-<01:02.600><i>Five.</i>
+Four.
+No. Wait yes!
+Five.
```

Comment:
I feel like we need _some_ sort of differential between talking people, not sure what